src/frontends/pytorch/src/op/rad2deg.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <cmath>
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/multiply.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_rad2deg(const NodeContext& context) {
+    // Ensure that the operation has exactly one input (the input tensor)
+    num_inputs_check(context, 1, 1);
+
+    // Retrieve the input tensor
+    auto input = context.get_input(0);
+
+    // Get the input element type dynamically
+    auto input_type = input.get_element_type();
+
+    const double pi_val = std::atan(1.0) * 4;
+
+    // Create a constant node with the conversion factor (180 / π)
+    auto conversion_factor = context.mark_node(ov::op::v0::Constant::create(input_type, Shape{}, {180.0 / pi_val}));
```

Comment:
Do not use input element_type to create a constant. The input type can be unknown (dynamic), create constant with fp64 type and use ConvertLIke to convert constant to required type.